packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp

```diff
@@ -327,42 +327,6 @@ const Epetra_IntMultiVector & toEpetra(const MultiVector<int, int, GlobalOrdinal
       // do nothing
     }
 
-
-    //! @name Xpetra specific
-    //@{
-#ifdef HAVE_XPETRA_KOKKOS_REFACTOR
-    typedef typename Xpetra::MultiVector<Scalar, LocalOrdinal, GlobalOrdinal, Node>::dual_view_type dual_view_type;
-
-    typename dual_view_type::t_host_um getHostLocalView () const {
-      throw std::runtime_error("EpetraIntVector does not support device views! Must be implemented extra...");
-#ifndef __NVCC__ //prevent nvcc warning
-      typename dual_view_type::t_host_um ret;
-#endif
-      TEUCHOS_UNREACHABLE_RETURN(ret);
-    }
-
-    typename dual_view_type::t_dev_um getDeviceLocalView() const {
-      throw std::runtime_error("Epetra does not support device views!");
-#ifndef __NVCC__ //prevent nvcc warning
-      typename dual_view_type::t_dev_um ret;
-#endif
-      TEUCHOS_UNREACHABLE_RETURN(ret);
-    }
-
-    template<class TargetDeviceType>
-    typename Kokkos::Impl::if_c<
-      std::is_same<
-        typename dual_view_type::t_dev_um::execution_space::memory_space,
-        typename TargetDeviceType::memory_space>::value,
-        typename dual_view_type::t_dev_um,
-        typename dual_view_type::t_host_um>::type
-    getLocalView () const {
-      return this->MultiVector< Scalar, LocalOrdinal, GlobalOrdinal, Node >::template getLocalView<TargetDeviceType>();
-    }
-#endif
-
-    //@}
-
```

Comment:
Is this not tested in MueLu?  We pass all of them, I'm pretty sure this will call the MutliVector call...

Reply:
I can put this back but looking at the code I don't see why the call here which just calls the multivector instance is any different than calling the one from the class hierarchy defined here

  class EpetraIntMultiVectorT
    : public MultiVector<int,int,EpetraGlobalOrdinal, Node>

Since MultiVector<> has a concrete def of getLocalView it will be defined without the reference.

Let me know if you want the explicit nature of this but my feeling is if we can compile the full stack and the tests pass then it isn't needed.  I'm close to building EMPIRE and that will test it as well